packages/recordio/src/read.js

```diff
@@ -0,0 +1,25 @@
+export default function read(input) {
+  const records = [];
+
+  while (/^\d+\n.+/.test(input)) {
```

Comment:
Please _cache_ the regular expression to not re-instantiate the expression over and over again.